build/gulpfile.hygiene.js

```diff
@@ -223,6 +223,11 @@ const tslintHygieneFilter = [
 	...tslintBaseFilter
 ];
 
+const fileLengthFilter = filter([
+	'!extensions/import/*.docx',
```

Comment:
The filter needs to be 
```
const fileLengthFilter = filter([
	'**',
	'!extensions/import/*.docx',
	'!extensions/admin-tool-ext-win/license/**'
], {restore: true});
```

Because right now without including anything you're filtering out all the files. 

Good find on the restore for the filter though - that's super cool!

Reply:
Updated it, thanks!